<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Particulas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Agregar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Particulas</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="7" column="1">
            <widget class="QSpinBox" name="Velocidad_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Id:</string>
             </property>
            </widget>
           </item>
           <item row="12" column="0" colspan="2">
            <widget class="QPushButton" name="agregar_final_pushButton">
             <property name="text">
              <string>Agregar final</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Origen</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string> X:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string> Y:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="Origen_X_spinBox">
                <property name="maximum">
                 <number>500</number>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QSpinBox" name="Origen_Y_spinBox">
                <property name="maximum">
                 <number>500</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Velocidad:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="Id_lineEdit"/>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>Color</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>R:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="Red_spinBox">
                <property name="maximum">
                 <number>255</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>G:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="Green_spinBox">
                <property name="maximum">
                 <number>255</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>B:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="Blue_spinBox">
                <property name="maximum">
                 <number>255</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="5" column="0" colspan="2">
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Destino</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>X:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="Destino_X_spinBox">
                <property name="maximum">
                 <number>500</number>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Y:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QSpinBox" name="Destino_Y_spinBox">
                <property name="maximum">
                 <number>500</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="11" column="0" colspan="2">
            <widget class="QPushButton" name="agregar_inicio_pushButton">
             <property name="text">
              <string>Agregar inicio</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>Salida</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPlainTextEdit" name="salida"/>
           </item>
           <item>
            <widget class="QPushButton" name="mostrar_pushButton">
             <property name="text">
              <string>Mostrar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ordenar_id_pushButton">
             <property name="text">
              <string>Ordenar Id</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ordenar_distancia_pushButton">
             <property name="text">
              <string>Ordenar Distancia</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ordenar_velocidad_pushButton">
             <property name="text">
              <string>Ordenar Velocidad</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tabla</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0" colspan="3">
         <widget class="QTableWidget" name="tabla"/>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="buscar_lineEdit">
          <property name="placeholderText">
           <string>Id de la particula</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="buscar_pushButton">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="mostrar_tabla_pushButton">
          <property name="text">
           <string>Mostrar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Dibujar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="2" column="1">
         <widget class="QPushButton" name="limpiar">
          <property name="text">
           <string>Limpiar</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="dibujar">
          <property name="text">
           <string>Dibujar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>599</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
   </widget>
   <widget class="QMenu" name="menuVista">
    <property name="title">
     <string>Vista</string>
    </property>
    <addaction name="actionPuntos"/>
   </widget>
   <widget class="QMenu" name="menuAlgoritmos">
    <property name="title">
     <string>Algoritmos</string>
    </property>
    <addaction name="actionPuntos_mas_Cercanos"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuVista"/>
   <addaction name="menuAlgoritmos"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPuntos">
   <property name="text">
    <string>Puntos</string>
   </property>
  </action>
  <action name="actionPuntos_mas_Cercanos">
   <property name="text">
    <string>Puntos mas Cercanos</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
